src/components/ListMovementDonut.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  ArcElement,
+  Title,
+  Tooltip,
+  Legend,
+} from "chart.js";
+import { Doughnut } from "react-chartjs-2";
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  ArcElement,
+  Title,
+  Tooltip,
+  Legend
+);
+
+function generateRandomColors() {
```

Comment:
Lets standardize the coloring across all of the components, is there a way to pull it into a separate styling file for all of our graphical components?